gradle.properties

```diff
@@ -32,4 +32,4 @@ apachePassWord=
 signEnabled=false
 
 org.gradle.warning.mode=none
-org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
Did you increase the memory allocation here only after encountering out-of-memory issues during usage?

Reply:
Yes, the pmd task seems to consume a lot of memory space.